.bazelrc

```diff
@@ -62,6 +62,9 @@ build:asan --copt -D__SANITIZE_ADDRESS__
 build:asan --test_env=ASAN_OPTIONS=handle_abort=1:allow_addr2line=true:check_initialization_order=true:strict_init_order=true:detect_odr_violation=1
 build:asan --test_env=UBSAN_OPTIONS=halt_on_error=true:print_stacktrace=1
 build:asan --test_env=ASAN_SYMBOLIZER_PATH
+# ASAN needs -O1 to get reasonable performance.
+build:asan --copt -O1
+build:asan --copt -foptimize-sibling-calls
```

Comment:
If you're following https://releases.llvm.org/10.0.0/tools/clang/docs/AddressSanitizer.html, shouldn't this be `-fno-optimize-sibling-calls`?

Reply:
:facepalm: https://github.com/envoyproxy/envoy/pull/13691